virtaal/views/widgets/textbox.py

```diff
@@ -329,15 +331,16 @@ def move_elem_selection(self, offset):
     def __color_selector_textboxes(self, *args):
         """Put a highlighting border around the current selector text box."""
         if not hasattr(self, 'selector_color'):
-            self.selector_color = gtk.gdk.color_parse(current_theme['selector_textbox'])
+            self.selector_color = Gdk.color_parse(current_theme['selector_textbox'])
         if not hasattr(self, 'nonselector_color'):
-            self.nonselector_color = self.parent.style.bg[gtk.STATE_NORMAL]
+            self.nonselector_color = self.get_parent().get_style_context().get_color(Gtk.StateType.NORMAL)
```

Comment:
Is this correct? Doesn't .get_color() give the foreground colour?

Reply:
this is wrong.get_color() give the forground color. It should be get_background_color